### PR TITLE
IPA-719 Add initial files. Unit test runs fine.

### PR DESCRIPTION
**Pull request by shirleyqiantingliu**
# Problem / Feature
Add initial files for the component.

# Changes
At least unit test can run.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+/dist
+/docs
+node_modules/
+.nyc_output/
+coverage/
+test-results.xml
+junit-results.xml
+lint-results.xml
+blob.env
+email.env
```
Comment (huukias) [inline .gitignore]: Could we need .npmrc also here at some point?

Comment (shirleyqiantingliu) [reply .gitignore]: We can add later.

File: config/test.setup.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const { JSDOM } = require('jsdom');
+// const { createDatabase } = require('./database');
```
Comment (huukias) [inline config/test.setup.js]: Why do we have commented code in this file?

Comment (shirleyqiantingliu) [reply config/test.setup.js]: It can be added when we want to use database test for unit test.

**Comment by shirleyqiantingliu** [comment]
I did not include every changes we need in PR. My idea is as far it does not break other one's work, we could commit as small/often as it possible.

**Pull request merged**